openedx/core/djangoapps/credit/tests/test_verification_access.py

```diff
@@ -80,8 +80,9 @@ def test_creates_user_partitions(self):
         self._update_partitions()
 
         # Check that a new user partition was created for the ICRV block
-        self.assertEqual(len(self.course.user_partitions), 1)
-        partition = self.course.user_partitions[0]
+        # One partition (the enrollment track partition) already exists.
+        self.assertEqual(len(self.course.user_partitions), 2)
```

Comment:
Christina: change test to return non-enrollment track partition (like test above).